tests/UI/pages/BO/shipping/carriers/add.js

```diff
@@ -48,8 +48,8 @@ class AddCarrier extends BOBasePage {
 
   /**
    * Fill carrier form in create or edit page and save
-   * @param page
-   * @param carrierData
+   * @param page {Page} Browser tab
+   * @param carrierData {Object} Carrier information
```

Comment:
```suggestion
   * @param carrierData {Object} Carrier information
```

I know I wasn't complaining on this before, but Object is same as considering it as empty => {} is an object, but createEditCarrier couldn't work with `{}` as value, as it's just JSDoc, it won't change anything but it would be `{CarrierData}` or `{{firstAttr: String, secondAttr: Int...}}`

Reply:
Hi @NeOMakinG thanks for the review as seen with @boubkerbribri we will privilege the option {CarrierData} rather than noting all the attributes .